src/renderer/components/+catalog/catalog.tsx

```diff
@@ -92,24 +92,34 @@ export class Catalog extends React.Component {
   }
 
   @action
-  onTabChange = (tabId: string) => {
-    this.activeTab = tabId;
+  onTabChange = (tabId: string | null) => {
+    const activeCategory = this.categories.find(category => category.getId() === tabId);
 
-    const activeCategory = this.categories.find((category) => category.getId() === tabId);
-
-    if (activeCategory) {
-      this.catalogEntityStore.activeCategory = activeCategory;
-    }
+    this.catalogEntityStore.activeCategory = activeCategory;
+    this.activeTab = activeCategory?.getId();
   };
 
   renderNavigation() {
     return (
       <Tabs className="flex column" scrollable={false} onChange={this.onTabChange} value={this.activeTab}>
         <div className="sidebarHeader">Catalog</div>
         <div className="sidebarTabs">
-          { this.categories.map((category, index) => {
-            return <Tab value={category.getId()} key={index} label={category.metadata.name} data-testid={`${category.getId()}-tab`} />;
-          })}
+          <Tab
+            value={null}
+            key="*"
+            label="Browse"
```

Comment:
Or
```suggestion
            label="All Categories"
```

Can this Tab be visually differentiated from the actual Category tabs? Also, it should highlight like the individual categories do.

Reply:
I've seen the mockup, it has differentiation from the categories, but `"Browse"` still seems odd to me. If there's a nested list under CATEGORIES as in the mockup then we might as well support multi-select of the categories, and selecting CATEGORIES acts as select-all? Or checkboxes, with a select-all checkbox next to CATEGORIES?